frontend/src/CollectionForm/CollectionForm.tsx

```diff
@@ -0,0 +1,170 @@
+import {
+  createStyles,
+  InputAdornment,
+  InputBase,
+  makeStyles,
+  Snackbar,
+  TextField,
+  Theme
+} from "@material-ui/core";
+import axios from "axios";
+import React, { SyntheticEvent, useEffect, useState } from "react";
+import EditIcon from '@material-ui/icons/Edit';
+import Autocomplete from "@material-ui/lab/Autocomplete";
+import ActionButtons from "../ActionButtons/ActionButtons";
+import SearchCard from "SideBarSearch/searchCard";
+import Alert from "@material-ui/lab/Alert";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    flexContainer: {
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "start",
+      width: "40vw",
+      maxWidth: "500px",
+      padding: theme.spacing(3)
+    },
+    field: {
+      width: "100%",
+      margin: theme.spacing(0, 0, 2, 0)
+    },
+    title: {
+      fontSize: "180%",
+    },
+  })
+);
+
+function CollectionForm() {
+  const [murals, setMurals] = useState<any>([]);
+  const [muralsInCollection, setMuralsInCollection] = useState<any>([]);
+  const [muralResults, setMuralResults] = useState<any>([]);
+  const [muralQuery, setMuralQuery] = useState<string>("");
+
+  const [description, setDescription] = useState([]);
+  const [editingDesc, setEditingDesc] = useState<boolean>(false);
+  const [hoveringDesc, setHoveringDesc] = useState<boolean>(false);
+
+  const [title, setTitle] = useState([]);
+  const [editingTitle, setEditingTitle] = useState<boolean>(false);
+  const [hoveringTitle, setHoveringTitle] = useState<boolean>(false);
+
+  const [popup, setPopup] = useState<boolean>(false);
+
+  const styles = useStyles();
+
+  const handleAddMural = (addedMural: any) => {
+    if (!addedMural) return
+    if (muralsInCollection.some((currMural: any) => currMural.id === addedMural.id))
+      return
+
+    setMuralsInCollection((oldMurals: any) => [...oldMurals, addedMural]);
+  };
+
+  const handleSave = () => {
+    if (!title.length || !description.length) return;
+
+    axios.post('http://localhost:3000/collection', {
```

Comment:
URLs should be parametrized or in constants

Reply:
I'm going to leave these for now for issue #101